lib/pkcs11h-core.c

```diff
@@ -758,12 +1091,7 @@ pkcs11h_addProvider (
 		goto cleanup;
 	}
 
-	memset(&initargs, 0, sizeof(initargs));
-	if ((initargs.pReserved = getenv("PKCS11H_INIT_ARGS_RESERVED")) != NULL) {
```

Comment:
please preserve this functionality.
consider adding this as property, as if we implement configuration file we might need it.

Reply:
I decide to remove this because: 
1. it's can be used as property by setting init_args. 
2. PKCS#11 says:
>  pReserved       reserved for future use.  Should be NULL_PTR for this version of Cryptoki
3. Using of enviroment variables is a bad approach. Pkcs11-helper may use sevaral libs with differ pReserved.

I try to add setting of this using config file. But what if pReserved will be not only ```char *``` in the future PKCS#11 standard?